samples/node/ConsoleApp/README.md

```diff
@@ -0,0 +1,21 @@
+# Node Console App
+This sample demonstrates a simple console app to be run in node. 
+
+Restore dependencies
+--------------------
+```
+npm install
+dotnet restore
+```
+Build the app
+-------------------
+This will start the Fable server and invoke webpack to build and bundle the app.
+```
+dotnet fable npm-run build
```

Comment:
Webpack's dead code elimination *suuuuuuuucks*.

Rollup is simpler than Webpack, and the bundle output is **much** smaller. It's just that Webpack has more features than Rollup, and Webpack is more popular than Rollup.